runtime/tutor

```diff
@@ -1340,6 +1340,182 @@ actions available in a popup. This will allow you to:
+specific split.
+
+You can also close all splits except the current one with Ctrl-w o .
+Open a third vertical split with Ctrl-w nv , then move to the
+leftmost split with Ctrl-w h twice, then from inside the split on
+the left press Ctrl-w o to close all except this split.
+
+
+
+=================================================================
+=               CHAPTER 13.3 SPLIT CURRENT BUFFER               =
+=================================================================
+
+You can also split the view of the current buffer: use Ctrl-w
+s (to split half way the current buffer horizontally and open
+a copy of it on the right too) and again use Ctrl-w v (to split
+half way vertically with the buffer opened on both splits).
+
+Close extra splits with Ctrl-w o to come back to a single window.
+
+
+
+
+
+
+
+
+
+
+
+
+=================================================================
+=               CHAPTER 13.4 USE COMMANDS TO SPLIT              =
+=================================================================
+
+You can also use the :vsplit (aliased to :vs for convenience) or
+the :hsplit (aliased to :hs ) to split vertically or horizontally
+a specific buffer. For example, type in the command:
+
+:vs something
+
+to open a new vertical split called something to the right. Here
+something is not an existing file, so this will open a new buffer
+under this name, but you could replace something with any file name
+to open it in a new buffer. Similarly, you can type the command:
+
+:hs some_more
+
+to open a new buffer called some_more in the lower half.
+some_more could be any file or path to open this specific file or path
+instead of a new empty buffer.
+
+
+=================================================================
+=                  CHAPTER 13.5 SWAPPING SPLITS                 =
+=================================================================
+
+Open a split on the left with :vs hello1 and then a split below
+with :hs hello2 .
+
+From hello2, press Ctrl-w K to swap with the split above. Now the
+split hello2 is on the top while hello1 is on the bottom.
+
+Still from hello2, press Ctrl-w H to swap with the split on the
+left: now, hello2 is on the left, and the tutor is on the top
+right. After Ctrl-w, you can use HJKL to split with the buffer
+on the left / below / above / on the right.
+
+Move back to the tutor split, and press Ctrl-w o to only keep
+this split.
+
+
+
+
+
+=================================================================
+=                CHAPTER 13.6 TRANSPOSE SPLITS                  =
+=================================================================
+
+Open a split on the left with :vs hello1 and then a split below
+with :vs hello2 .
+
+Move to the tutor split, then press Ctrl-w t to transpose the
+vertical split that was opened from this window: now hello1 and
+hello2 and below, rather than on the right, of the tutor. Press
+Ctrl-w t to transpose back.
```

Comment:
```suggestion
Move to the tutor split, then press Ctrl-w t to transpose the
vertical split opened from this window: now, hello1 and
hello2 are below, rather than to the right of, the tutor. Press
Ctrl-w t again to transpose back.
```